cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -248,13 +247,26 @@ public Path getMavenConfFolder(boolean legacy) {
    * @return the maven arguments (MVN_ARGS).
    */
   public String getMavenArgs() {
-    Path mavenConfFolder = getMavenConfFolder(false);
+    Path mavenConfFolder = getMavenConfFolder(true);
```

Comment:
While reverting your change, you got the flag changed. IMHO by accident and not on purpose.
```suggestion
    Path mavenConfFolder = getMavenConfFolder(false);
```